bot/exts/moderation/clean.py

```diff
@@ -285,15 +285,17 @@ async def _delete_found(self, message_mappings: dict[TextChannel, list[Message]]
 
                 if len(to_delete) == 100:
                     # Only up to 100 messages can be deleted in a bulk
-                    await channel.delete_messages(to_delete)
+                    with suppress(NotFound):
+                        await channel.delete_messages(to_delete)
```

Comment:
I believe in this case we can leave out the `suppress`, as we know it will be a bulk delete per `len(to_delete) == 100`, which doesn't care that the messages are absent.

Reply:
I agree.

![image](https://user-images.githubusercontent.com/75779179/143271440-e34b3bc0-0a5d-44c2-99fd-3e816bc37d17.png)
My bad I missed, it was specified there.